### ISSUE TITLE
Pattern gallery not showing the patterns

### ISSUE THREAD

**Issue by ruubiiiii** [OliBomby/Mapping_Tools#208]
## Prerequisites

- [ x] I have read the [Mapping Tools FAQ](https://mappingtools.github.io/faq)

## Environment

**OS version**: Windows 10

**Mapping Tools version**: 1.12.19.0

## Description

So basically the issue is when you copy sliders or some patterns in the pattern gallery it only shows the picture of it at that time, when you restart the app it becomes grey. 

![image](https://github.com/user-attachments/assets/dd339ac3-346b-4a8f-8301-366c1104b403)

Here is an example of the issue

**Comment by OliBomby** [comment]
I'm unable to recreate the issue for myself.

Can you still export the patterns whose picture is grey? Do you see a file if you right-click them and 'Open in file explorer'?

**Comment by ruubiiiii** [comment]
yes you can but it makes it extra hard to find the pattern that you want

**Comment by OliBomby** [comment]
Hmm if the pattern file still exists and it can read it, then I have no clue what could be the issue...
